image.yaml

```diff
@@ -10,6 +10,7 @@ ignition-network-kcmdline: []
 
 ostree-format: "oci"
 
+vmware-secure-boot: "false"
```

Comment:
Let's add a comment here with a little context/warning about changing this value